charts/spire/charts/spiffe-oidc-discovery-provider/values.yaml

```diff
@@ -215,3 +215,16 @@ tests:
       version: ""
       # -- Overrides the image tag
       tag: 5.2.15
+
+deleteHook:
```

Comment:
It would not require a library chart change. Just a 3 line changes. change deleteHook here to:
```
tools:
  kubectl
```

and in in the hook, image:   update the value to tools.kubectl